test/e2e/testnet/test_helpers.go

```diff
@@ -1,11 +1,87 @@
 package testnet
 
 import (
+	"context"
+	"encoding/json"
+	"fmt"
 	"log"
+	"strconv"
+	"time"
+
+	"github.com/celestiaorg/knuu/pkg/knuu"
 )
 
 func NoError(message string, err error) {
 	if err != nil {
 		log.Fatalf("%s: %v", message, err)
 	}
 }
+
+type JSONRPCError struct {
+	Code    int
+	Message string
+	Data    string
+}
+
+func (e *JSONRPCError) Error() string {
+	return fmt.Sprintf("JSONRPC Error - Code: %d, Message: %s, Data: %s", e.Code, e.Message, e.Data)
+}
+
+func getStatus(executor *knuu.Executor, app *knuu.Instance) (string, error) {
```

Comment:
Where does this get used?

Reply:
can be removed when https://github.com/celestiaorg/celestia-app/pull/3505 is merged